### ISSUE TITLE
Installation on 64-bit Debian

### ISSUE THREAD

**Issue by DMJC** [ChristophHaag/SteamVR-OpenHMD#4]
Hi I thought this should be put up here so People can work around a fault in Steam I found.

**Comment by ChristophHaag** [comment]
Right, that's Valve's "glue" library that provides the openvr API on top of SteamVR.

Personally I have the library from https://github.com/ValveSoftware/openvr/tree/master/lib/linux64 installed in /usr/lib so for me this wasn't a problem.

I think tomorrow I'll just add small scripts to the repository to add and remove SteamVR-OpenHMD from steamvr (and save and restore steamvr.vrsettings), and that uses LD_LIBRARY_PATH for libopenvr_api.so
